test/JDBC/expected/parallel_query/with_recompile-vu-verify.out

```diff
@@ -0,0 +1,3450 @@
+SELECT set_config('babelfishpg_tsql.explain_costs', 'off', false)
+go
+~~START~~
+text
+off
+~~END~~
+
+
+
+
+
+/*
+ * These tests validate the T-SQL RECOMPILE behaviour for stored procedures.
```

Comment:
Since we change `plan_cache_mode` GUC during executions, the following cases must be tested:
1. What happens if a user changes `plan_cache_mode` inside a procedure?
2. Is `plan_cache_mode` restored properly after each execution?
3. Is `plan_cache_mode` restored properly even if an exception is thrown?

Reply:
A test case was added which set the `plan_cache_mode` inside the proc, and another one where an exception is triggered after that.  In both cases, the `plan_cache_mode` is correctly reset to the value before the proc was called. Look for `p_recomp_71` and `p_recomp_72`.